actfw_core/capture.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, device='/dev/video0', size=(640, 480), framerate=30,
             framerate (int): expected capture framerate
             expected_format (:class:`~actfw_core.v4l2.video.V4L2_PIX_FMT`): expected capture format
             fallback_formats (list of :class:`~actfw_core.v4l2.video.V4L2_PIX_FMT`): fallback capture format
+            format_selector (:class:`~actfw_core.capture.V4LCameraCapture.FormatSelector): DEFAULT selects the first format. PROPER selects the most proper format. MAXIMUM selects the largest resolution (and ignores framerate) format as a camera can.
```

Comment:
To emphasize framerate w/ MAXIMUM option:

```suggestion
            format_selector (:class:`~actfw_core.capture.V4LCameraCapture.FormatSelector): DEFAULT selects the first format. PROPER selects the most proper format. MAXIMUM selects the largest resolution format as a camera can. **MAXIMUM ignores framerate parameters (uses appropriate framerate for the selected resolution).**
```

Reply:
> DEFAULT selects the first format. PROPER selects the most proper format.

Users may not understand what are `first` / `proper`.
Could elaborate more?

Reply:
I'll try

Reply:
https://github.com/Idein/actfw-core/pull/4/commits/51303ed3bda513409bf592ce95e8d69006fb0e83